aidial_assistant/chain/dialogue.py

```diff
@@ -1,17 +1,27 @@
-from aidial_assistant.chain.model_client import Message
+from pydantic import BaseModel
+
+from aidial_assistant.model.model_client import Message
+
+
+class Exchange(BaseModel):
```

Comment:
How about to call it `DialogueTurn`?

I borrowed it from the [Bison spec](https://cloud.google.com/vertex-ai/docs/generative-ai/learn/models), which specifies max number of *turns*.